src/components/dashboard/sidebar/CouponCodesSummaryCard.jsx

```diff
@@ -70,7 +78,13 @@ const CouponCodesSummaryCard = ({
           cardClassNames={className}
         >
           <p className="m-0">
-            <h3 className="float-left"> {couponCodesCount > 0 ? couponCodesCount : 0}</h3>{' '}<span className="ml-2">{COUPON_CODES_SUMMARY_DETAIL}</span>
+            <h3 className="float-left"> {couponCodesCount > 0 ? couponCodesCount : 0}</h3>{' '}
+            <span className="ml-2">{intl.formatMessage({
```

Comment:
I think this can be written using plural format. Example below.

```python
{
  intl.formatMessage({
    id: 'enterprise.dashboard.sidebar.coupon-codes.summary.title',
    defaultMessage: 'Enrollment Codes{couponCodesCount, plural, =0 {} other {: #}}',
    description: 'Title for the coupon codes summary on the enterprise dashboard sidebar.',
  },
    {
      couponCodesCount: 10
    }
  )
}
```